src/app/components/AccountMenu/index.tsx

```diff
@@ -193,8 +196,10 @@ function AccountMenu({ showOptions = true }: Props) {
                   openOptions("accounts/new");
                 }}
               >
-                <PopiconsPlusLine className="h-4 w-4 mr-2 shrink-0" />
-                {t("options.account.add")}
+                <PopiconsCirclePlusLine className="h-4 w-4 mr-2 shrink-0 text-gray-800 dark:text-neutral-200" />
```

Comment:
I know it's in the Figma like that, but I think both of the elements should have the same color, possibly just the default ones. (so you could remove all color classes here and below)